src/Parser/BloodDeathKnight/Modules/Features/CastEfficiency.js

```diff
@@ -56,26 +55,19 @@ class CastEfficiency extends CoreCastEfficiency {
       spell: SPELLS.DANCING_RUNE_WEAPON,
       category: CastEfficiency.SPELL_CATEGORIES.COOLDOWNS,
       getCooldown: haste => 180,
-      recommendedCastEfficiency: 0.75,
+      recommendedCastEfficiency: 0.85,
       extraSuggestion: <span>Should be used as an openner and used on CD for the dps boost.</span>,
     },
 
     {
       spell: SPELLS.BLOODDRINKER,
       category: CastEfficiency.SPELL_CATEGORIES.ROTATIONAL,
       getCooldown: haste => 30,
-      recommendedCastEfficiency: 0.85,
+      recommendedCastEfficiency: 0.90,
       isActive: combatant => combatant.hasTalent(SPELLS.BLOODDRINKER_TALENT.id),
       extraSuggestion: <span>Mostly used as a dps CD. Should be almost casted on CD. Good to use when your running to the boss or cant melee them.</span>,
     },
 
-
-    {
```

Comment:
Why was this removed? Using active racials like Arcane Torrent generate resources that can be used to improve play, using it on cooldown should be trivial.